authnid/make_app.py

```diff
@@ -61,6 +61,9 @@ def _make_crl_validator(app):
     app.crl_validator = Validator(
         roots=[app.config["CA_CHAIN"]], crl_locations=crl_locations
     )
+    if len(app.crl_validator.errors) > 0:
```

Comment:
I think this block (64 - 66) can be simplified by removing the `if` line.

Reply:
yup, good catch.